src/FakeItEasy/JetBrainsAnnotationsJustificationMessages.cs

```diff
@@ -0,0 +1,15 @@
+namespace FakeItEasy
+{
+    using JetBrains.Annotations;
+
+    /// <summary>
+    /// Common justification messages for JetBrains ReSharper annotations.
+    /// </summary>
+    public static class JetBrainsAnnotationsJustificationMessages

```

Comment:
Should probably be `internal`, like the attribute. There's no need to expose this.